src/lib/events/eventTextDialogue.js

```diff
@@ -9,9 +9,17 @@ export const fields = [
     type: "textarea",
     maxPerLine: 18,
     placeholder: l10n("FIELD_TEXT_PLACEHOLDER"),
-    updateFn: trimlines,
+    updateFn: (string, field) => trimlines(string, field.maxPerLine),
```

Comment:
I can't find a simple way to enforce the 16 characters per line limit when an avatar is added. @chrismaltby do you have any suggestion?